bears/generate_package.py

```diff
@@ -120,6 +121,17 @@ def main():
             bear_name, _ = os.path.splitext(os.path.basename(bear_file_name))
             create_file_structure_for_packages(
                 os.path.join('bears', 'upload'), bear_file_name, bear_name)
+            if bear_object.REQUIREMENTS:
+                for requirement in bear_object.REQUIREMENTS:
+                    if isinstance(requirement, PipRequirement):
+                        with open(os.path.join(
+                                    'bears', 'upload',
+                                    bear_name, 'requirements.txt'),
+                                  'a') as reqtxt:
```

Comment:
because else if you have 3 requirements you would overwrite them for every requirement and end up only with the last one?